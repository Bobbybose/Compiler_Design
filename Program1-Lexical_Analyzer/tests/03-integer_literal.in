1
12
123
01
101
00123