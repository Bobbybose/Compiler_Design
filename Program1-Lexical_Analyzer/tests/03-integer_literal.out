integer-literal: 1: 1, 1
integer-literal: 12: 2, 1
integer-literal: 123: 3, 1
integer-literal: 01: 4, 1
integer-literal: 101: 5, 1
integer-literal: 00123: 6, 1